### PR TITLE
initial thoughts and feedback - DO NOT MERGE

### PR DESCRIPTION
**Pull request by bayanijulian**
Interesting project, are there any specific use cases you are targetting? For myself, during leetcode style interviews, anytime there was a question dealing with binary/byte manipulation (shifting operators) I would immediately switch from python to c. Then I lose out on maps and lists and easy string functions. In C its nice to know the exact size of bytes variables take up or creating buffers on the fly with chars[]

I'm still confused on what the syntax is or the language spec really looks like. Listing out the keywords of the language would be helpful or creating a DFA/CFA diagram. Reading the sample at the end of the README gives me golang vibes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,7 +175,7 @@ Obj Person:
         Int Age = _Age
```
Comment (bayanijulian) [inline README.md:175]: To clarify, are you saying objects only have a single constructor. Or why do we need this and if we do, why are no types specified?  
```
 \ There are no internal constructors or methods for objects \
 Fnc New(_Name, _Age)
        Str Name = _Name
        Int Age = _Age

```

File: README.md [hunk 2]
```diff
@@ -145,14 +148,14 @@ While Number < 15
...
-    "Strawberry",
+    "Strawberry", (trailing comma mandatory?)
 ]
 
 For Fruit in Fruits
     Out(Fruit)
 
 \ Function Declaration: Read_File \
-Fnc Read_File(String FilePath) -> Rtn
+Fnc Read_File(String FilePath) -> Rtn (is Rtn None? or void)
```
Comment (bayanijulian) [inline README.md:158]: Is Rtn None/void?

File: README.md [hunk 3]
```diff
@@ -121,6 +121,9 @@ comment in
 Pineapple
 \ 
 
+\\\ Invalid Inline Comment
```
Comment (bayanijulian) [inline README.md:124]: Is it better to say there no inline comments and only multiline comments? I'm just trying to think of a language where inline comments requires an EOL token - only language that comes to mind is HTML

File: README.md (hunk 2 above)
Comment (Robert-DeForrest-Reynolds) [reply README.md:158]: `Rtn` indeed simply returns "None" which is simply a constant that represents void, usually specifically a void pointer.

File: README.md (hunk 3 above)
Comment (Robert-DeForrest-Reynolds) [reply README.md:124]: This is just wrongly worded, apologies, I'll fix this. All comments are simply wrapped with two backslashes `\ Comment \`

File: README.md (hunk 1 above)
Comment (Robert-DeForrest-Reynolds) [reply README.md:175]: There are no constructors at all by default. This is an example of implementing a constructor method manually. In Python you have `__init__` overwriting, or in C# creating a method of the same name of the class. In Pineapple, there is nothing built-in to construct an object. You can either specify properties at instantiation, or create a constructor method.

**Pull request closed without merging**